docs/en/changes/changes.md

```diff
@@ -81,6 +81,16 @@
 * Support displaying the port services listen to from OAP and UI during server start.
 * Refactor data-generator to support generating metrics.
 * Fix `AvgHistogramPercentileFunction` legacy name.
+* [Break Change] Labeled Metrics support multiple labels.
```

Comment:
OAL function change, function `percentile2` to replace`percentile` should be listed here.